src/fetch_tse_data.py

```diff
@@ -0,0 +1,254 @@
+import shutil
+import pandas as pd
+import numpy as np
+import os
+import urllib
+import zipfile
+import glob
+
+
+
+FILENAME_PREFIX='consulta_cand_'
+TEMP_PATH = '../data/tse_temp'
+TSE_CANDIDATES_URL='http://agencia.tse.jus.br/estatistica/sead/odsele/consulta_cand/'
+OUTPUT_DATASET_PATH = '../data/2017-03-31-tse-candidates.xz'
+os.makedirs(TEMP_PATH)
+
+# setting year range from 2004 to 2016. this will be modified further to 'from 1994 to 2016'
```

Comment:
Why do we generate years from 2004 to 2016 and, later, transform it? Wouldn't it worth it to start from 1994 right now?